src/NaadVars.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Bcgov\NaadConnector;
+
+/**
+ * Class NaadVars
+ *
+ * This class is responsible for managing environment variables used in the
+ * application. It retrieves values from the environment or uses default values
+ * when necessary. The class provides a dynamic way to access these variables
+ * through the magic __get and __set methods.
+ *
+ * Environment variables are essential for configuring the application without
+ * hardcoding sensitive information or settings. This class specifically handles
+ * database and application-related environment variables.
+ *
+ * @category Utility
+ * @package  Bcgov\NaadVars
+ * @author   Richard O'Brien <Richard.Obrien@gov.bc.ca>
+ * @license  https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link     https://alerts.pelmorex.com/
+ */
+class NaadVars
+{
+
+    private ?string $rootPassword;
+    private ?string $serviceHost;
+    private ?string $servicePort;
+    private ?string $databaseName;
+    private ?string $phpMyAdminPort;
+    private ?string $destinationURL;
+    private ?string $destinationUser;
+    private ?string $destinationPassword;
+    private ?string $logFilePath;
+    private ?string $naadName;
+    private ?string $naadUrl;
+    private ?string $naadRepoUrl;
+
+
+    /**
+     * Predefined environment variable keys and their defaults.
+     */
+    private static array $defaultValues = [
+        'logFilePath'  => '/var/www/html/naad-socket.log',
```

Comment:
```suggestion
        'logFilePath'  => 'naad-socket.log',
```
This whole env variable is going to be refactored away eventually anyway but for now this should be the default because otherwise I get this error when running start.php locally: `Fatal error: Uncaught UnexpectedValueException: There is no existing directory at "/var/www/html" and it could not be created: Permission denied`